From caf2a31d3947ddccff5f1660bce6ed8948007f9c Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Mon, 23 Jul 2012 10:24:58 +0800
Subject: [PATCH 1827/2463] ENGR00218013-1 DMA:Add HDMI DMA type and priv data
 for imx_dma_data

1 Add HDMI sdma periphal enum type.
2 Add private data type for imx_dma_data.

Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 arch/arm/plat-mxc/include/mach/dma.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/dma.h b/arch/arm/plat-mxc/include/mach/dma.h
index ba00959..3fccb3b 100644
--- a/arch/arm/plat-mxc/include/mach/dma.h
+++ b/arch/arm/plat-mxc/include/mach/dma.h
@@ -45,6 +45,7 @@ enum sdma_peripheral_type {
 	IMX_DMATYPE_IPU_MEMORY,	/* IPU Memory */
 	IMX_DMATYPE_ASRC,	/* ASRC */
 	IMX_DMATYPE_ESAI,	/* ESAI */
+	IMX_DMATYPE_HDMI,
 };
 
 enum imx_dma_prio {
@@ -58,6 +59,7 @@ struct imx_dma_data {
 	int dma_request_p2p;
 	enum sdma_peripheral_type peripheral_type;
 	int priority;
+	void *private;
 };
 
 struct imx_pcm_dma_params {
-- 
1.7.7.4

